values/gatekeeper/gatekeeper-raw.gotmpl

```diff
@@ -0,0 +1,7 @@
+{{- $v := .Environment.Values }}
+{{- $constraints := (exec "bash" (list "-c" "find  /tmp/otomi/constraints -name '*.yaml'")) | splitList "\n" }}
+resources:
+{{- range $constraint := $constraints }}{{ if ne $constraint "" }}
+{{/* < the splitList above results in empty lines in the array, so skipping them here */}}
```

Comment:
can this line stay appended to the previous line so it is clear what the comment is referring to?

Reply:
ok